pkg/validation/options.go

```diff
@@ -353,6 +353,12 @@ func parseSignatureKey(o *options.Options, msgs []string) []string {
 		return msgs
 	}
 
+	// `logger` isn't configured yet
+	_, err := fmt.Fprintln(os.Stderr, "WARNING: `--signature-key` is deprecated. It will be removed in a future release")
```

Comment:
In the PR I just merged, I moved the logging configuration to the top of the validation, so this should now be able to use logging, shall we update it?

Reply:
Nice! I'll rebase and make the shift 👍

Reply:
All set!